store/redis.go

```diff
@@ -46,6 +47,21 @@ func (s *RedisStore) Get(key interface{}) (interface{}, error) {
 	return s.client.Get(key.(string)).Result()
 }
 
+// GetWithTTL returns data stored from a given key and its corresponding TTL
+func (s *RedisStore) GetWithTTL(key interface{}) (interface{}, time.Duration, error) {
```

Comment:
I tried adding some tests for GetWithTTL but redis library is not so friendly when it comes to unit testing.

Reply:
Yes I understand it's not easy on this part ;)